### PR TITLE
adds the ability to move while buckled to office chairs

### PR DESCRIPTION
**Pull request by ghost**
k

still need to use that garbage rotate verb when you want to rotate but wasd/arrow keys work for movement

### CODE REVIEW

**Comment by pudl** [comment]
bay has it so you can rotate with wasd, do that

**Comment by paprka** [comment]
tried, not worth the effort, someone else can fuck with it

**Comment by TZK13** [comment]
Wouldn't somebody restrained to an office chair like this still be able to move? I'd understand opposition to that based on balance but it's not like people are typically cuffed to office chairs.

**Comment by dannno** [comment]
cool :+1:

**Comment by RemieRichards** [comment]
Might want to wait for https://github.com/tgstation/-tg-station/pull/7049

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 1]
```diff
@@ -128,6 +128,29 @@
...
+		buckled_mob.dir = dir
+		switch(buckled_mob.dir)
+			if(NORTH)
+				buckled_mob.dir = SOUTH
+			if(WEST)
+				buckled_mob.dir = EAST
+			if(SOUTH)
+				buckled_mob.dir = NORTH
+			if(EAST)
+				buckled_mob.dir = WEST
```
Comment (MrPerson) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm]: This should probably just be buckled_mob.dir = turn(buckled_mob.dir, 180)

**Comment by ike709** [comment]
gotta go fast

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm (hunk 1 above)
Comment (paprka) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm]: ah, didnt know that existed, thanks

**Comment by paprka** [comment]
@RemieRichards yes i will wait for that PR to go through and convert this to work with it so you dont have to, same with janicart

**Comment by paprka** [comment]
too lazy to wait for remie's thing nvm

**Pull request closed without merging**